packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -151,109 +227,205 @@ export const LivenessCameraModule = (
+            <RecordingIcon recordingIndicatorText={recordingIndicatorText} />
+          )}
+
+          {!isStartView && !isWaitingForCamera && !isCheckSucceeded && (
+            <CancelButton cancelLivenessCheckText={cancelLivenessCheckText} />
           )}
 
-          {/* 
+          <Overlay
+            anchorOrigin={{
+              horizontal: 'center',
+              vertical:
+                isRecording && !isFlashingFreshness ? 'start' : 'space-between',
+            }}
+            className={LivenessClassNames.InstructionOverlay}
+          >
+            <Hint hintDisplayText={hintDisplayText} />
+
+            {errorState && (
+              <ErrorView
+                onRetry={() => {
+                  send({ type: 'CANCEL' });
+                }}
+              >
+                {renderErrorModal({
+                  errorState,
+                  overrideErrorDisplayText: errorDisplayText,
+                })}
+              </ErrorView>
+            )}
+
+            {/* 
               We only want to show the MatchIndicator when we're recording
               and when the face is in either the too far state, or the 
               initial face identified state. Using the a memoized MatchIndicator here
               so that even when this component re-renders the indicator is only
               re-rendered if the percentage prop changes.
             */}
-          {isRecording &&
-          !isFlashingFreshness &&
-          showMatchIndicatorStates.includes(faceMatchState!) ? (
-            <MemoizedMatchIndicator
-              percentage={Math.ceil(faceMatchPercentage!)}
-            />
-          ) : null}
-        </Overlay>
-      </View>
-    </Flex>
+            {isRecording &&
+            !isFlashingFreshness &&
+            showMatchIndicatorStates.includes(faceMatchState!) ? (
+              <MemoizedMatchIndicator
+                percentage={Math.ceil(faceMatchPercentage!)}
+              />
+            ) : null}
+          </Overlay>
+
+          {isStartView &&
+            !isMobileScreen &&
+            selectableDevices &&
+            selectableDevices.length > 1 && (
+              <Flex className={LivenessClassNames.StartScreenCameraSelect}>
+                <View
+                  className={
+                    LivenessClassNames.StartScreenCameraSelectContainer
+                  }
+                >
+                  <Label
+                    htmlFor="amplify-liveness-camera-select"
+                    className={`${LivenessClassNames.StartScreenCameraSelect}__label`}
+                  >
+                    Camera:
+                  </Label>
+                  <SelectField
+                    id="amplify-liveness-camera-select"
+                    label="Camera"
+                    labelHidden
+                    value={selectedDeviceId}
+                    // eslint-disable-next-line @typescript-eslint/no-misused-promises
+                    onChange={onCameraChange}
```

Comment:
Can `onCameraChange` be treated as fire and forget?

Reply:
Just tested and yes it can, good call